### PR TITLE
Add --maximum-row-size option to zebra merge

### PR DESCRIPTION
**Pull request by jystic**
/jury approved @nhibberd

### CODE REVIEW

**Comment by tranma** [comment]
this is getting merge to drop big maps on the floor, right?
I'll savage zebra cat/export to output one entity.

**Comment by jystic** [comment]
i'm just experimenting

File: zebra-core/src/Zebra/Table/Logical.hs [hunk 1]
```diff
@@ -318,8 +350,7 @@ unionStep kvss =
         key =
           Boxed.minimum maximums
 
-        -- brexit --
-        (leaves0, nonvoters, remains) =
+        (done0, done1, incomplete) =
```
Comment (tranma) [inline zebra-core/src/Zebra/Table/Logical.hs]: haha

**Comment by jystic** [comment]
! @nhibberd

**Comment by nhibberd** [comment]
:+1: Yeah makes sense, tests probably be useful

**Pull request merged**